### DISCUSSION TITLE
Memoized Fields causes useFieldArray to put removed item back, once appended

### DISCUSSION THREAD

**Discussion by seleckis** [react-hook-form/react-hook-form#11195] [Q&A]
Here is my example: https://codesandbox.io/s/react-hook-form-usefieldarray-forked-6pjcdv?file=/src/index.js

1. Try to add a couple of records.
2. Remove one.
3. Append one more.

As a result, the removed items returns. But this happens only one time, after that items are removed and added as expected.

What's wrong with my code?

**Comment by tmnrp** [comment]
Seems like this issue asnwered ?
the code sanbox works fine according to me.

**Reply by seleckis** [reply]
Yes it works. I don't know why it wasn't.
